src/vos/vos_aggregate.c

```diff
@@ -1359,10 +1363,40 @@ unmark_removals(struct agg_merge_window *mw, const struct agg_phy_ent *phy_ent)
 	}
 }
 
+static void
+dump_trace(struct vos_agg_param *agg_param)
+{
+	vos_iter_entry_t *entry;
+	int               i;
+	int               last;
+
+	if (agg_param->ap_trace_count == 0)
+		return;
+
+	D_ERROR("Assertion will trigger, dumping the evt_trace\n");
+
+	if (agg_param->ap_trace_count <= EV_TRACE_MAX)
```

Comment:
According to L2202, it should be "if (agg_param->ap_trace_count < EV_TRACE_MAX)", otherwise, "i" cannot be equal to EV_TRACE_MAX, then the subsequent "while" will loop for ever.

Reply:
right, good catch.